soccer/motion/TrapezoidalMotionTest.cpp

```diff
@@ -3,17 +3,24 @@
 #include <Geometry2d/Point.hpp>
 #include <motion/TrapezoidalMotion.hpp>
 #include <math.h>
+#include <iostream>
 
+using namespace std;
 
 bool trapezoid1(float t, float &posOut, float &speedOut) {
```

Comment:
Let's make this a test case in the form TEST(TrapezoidalMotion, description).  EXPECT_* does'nt work inside regular functions.